contracts/evoting/evoting.go

```diff
@@ -101,6 +101,8 @@ func (e evotingCommand) createElection(snap store.Snapshot, step execution.Step)
 		ShuffleThreshold: threshold.ByzantineThreshold(roster.Len()),
 	}
 
+	PromElectionStatus.Set(float64(election.Status))
```

Comment:
To manage multiple concurrent elections, using a label would be necessary.
```suggestion
	PromElectionStatus.WithLabelValues(election.ElectionID).Set(float64(election.Status))
```

(here and elsewhere)